.gitlab-ci.yml

```diff
@@ -1,22 +1,74 @@
 stages:
```

Comment:
I wish we had a bit more documentation about how this change works and how it's getting used.

As someone working in the codebase, I have no idea why we need `host_inject.rb` and why it's different from `auto_inject.rb`, how and when we're releasing things as rpm/deb, and how to support customers than run into trouble with these.

Reply:
A `lib-injection/README.md` could help here^.